contracts/ReputationMiningCycle.sol

```diff
@@ -832,7 +832,7 @@ contract ReputationMiningCycle is PatriciaTreeProofs, DSMath {
       } else {
         // TODO: Is this safe? I think so, because even if there's over/underflows, they should
         // still be the same number.
-        require(int(agreeStateReputationValue)+logEntry.amount == int(disagreeStateReputationValue));
+        require(int(agreeStateReputationValue)+logEntry.amount == int(disagreeStateReputationValue), "colony-invalid-newest-reputation-proof");
```

Comment:
@area @elenadimitrova 

Stuck on something again... When I make this change, the migration `6_setup_meta_colony` step fails immediately.

```
6_setup_meta_colony.js
======================
Transaction: 0xbe82e5ff07add1ae1181106abc5f5824b0275da50c3dcb38e16ce2a6863f8a44 exited with an error (status 0). 
     Please check that the transaction:
     - satisfies all conditions set by Solidity `require` statements.
     - does not trigger a Solidity `revert` statement.
```

I'm not sure how adding this error message can cause step 6 to fail.

Any suggestions on how to get a more meaningful error message to debug this?

Reply:
It's failing on `startNextCycle` which I'm pretty sure has run out of gas :(
Without the error message, that call costs `6,582,315` gas and after you put it in it throws consuming all `6,721,975`. Although 140,000 seems like too much to add on by an extra error message. Continuing to investigate..

Reply:
Can you leave a todo there for now, summarising the above. This will likely need a separate piece of work altogether. Hopefully it won't trip you into adding most of the other `ReputationMiningCycle` error messages